localgovdrupal/localgov_forms
# Date styling

Recreating #20 with the correct commits.  Original author is @MattOz-CDS 


### Summary of change

We want inline validation error messages to appear above form input fields as per the GDS pattern
https://design-system.service.gov.uk/components/error-summary/

### To test
- Enable the LocalGov forms module
- Create a form with form elements that are required
- In the forms behaviour settings disable client-side validation
- Complete and submit a form with errors
- Verify that validation errors appear above the erroneous input field
